pkg/parse/parse.go

```diff
@@ -368,7 +368,7 @@ func getMounts(ctx *types.SystemContext, store storage.Store, mounts []string, c
 			finalMounts[mount.Destination] = mount
 			mountedImages = append(mountedImages, image)
 		case TypeCache:
-			mount, err := internalParse.GetCacheMount(tokens, store, "", nil)
+			mount, _, err := internalParse.GetCacheMount(tokens, store, "", nil)
```

Comment:
Discarding a list of locked files, which will need to be unlocked at some point, looks like a potential source of problems in the future.

Reply:
Pushed a version which takes care of this.